### ISSUE TITLE
Consider rename?

### ISSUE THREAD

**Issue by coleifer** [pysqlite3/pysqlite3#1]
I'm the maintainer of pysqlite3:

* https://github.com/coleifer/pysqlite3
* https://pypi.org/project/pysqlite3

I'm not sure quite why this fork exists, or how it differs from the above packages, but I think it would be considerate to rename this so that it is more clear:

1. Why this project exists separately from the above
2. How this project differs from the above
3. Could you rename this so that the differences are more obvious?

**Comment by nk9** [comment]
I would agree, it's quite confusing that this package is using such a similar name.

**Comment by edgarrmondragon** [comment]
I _think_ the main intention of this fork is publishing wheels for macOS and Windows: https://github.com/pysqlite3/pysqlite3/commit/b04ceb5fd11881b6843ee149d67ce72de273dee6
